docs/source/plugin/advanced.rst

```diff
@@ -64,3 +64,115 @@ a QUIT/PART event, and they will be available after a JOIN event.
 
 Note that you don't specifically need to use ``@plugin.thread(False)``, but
 it is still recommended to prevent any race condition.
+
+
+Capability Negociation
```

Comment:
```suggestion
Capability negotiation
```

Case change because it seems that we haven't used Title Case for this level of heading elsewhere in the page.